### ISSUE TITLE
[Feature request] Indicator of ajax request execution

### ISSUE THREAD

**Issue by orlov0562** [fguillot/kanboard#1793]
Hi there,

I am working with kanboard that installed on server  located at another continent, so when I do something I have small delays. Everything works great except one small thing - when I add/edit project or task and press [Save] button, few seconds looks like nothing happens. And only after 1-3 seconds the page refreshes. 

I understand that page wait for end of ajax request and just after completion it refresh the page. But may be make sense to show loading indicator or disable save button or smth like this, just to know that smth is going on.

If you don't understand what the problem is, just add ```sleep(5)``` to "new task" controller that serve ajax request and try to add new task.

**So, I suggest next feature:**
- when ajax request started
    - append "loading icon" to [save] button or before it
    - disable [save] button to avoid unwanted clicks/requests
- when ajax request stopped
   - remove "loading icon" or change icon to "complete icon" and hide it in few seconds
   - enable [save] button

**Comment by fguillot** [comment]
If you use the last version of KB, there is a loading icon for tasks.

**Comment by orlov0562** [comment]
Hello and thank you for fast response!

I have just updated my KB to v.1.0.25, and I still have not found any indicator. I have checked in FF and Chrome. I even install another copy to local, apend sleep(5) to index.php and still get same behaviour. Could you please explain in few words where indicator should located or give me css class or id of element that should appear?

If it helps I think I can record a desktop video with behaviour that I have and attach link to youtube to this thread.

**Comment by fguillot** [comment]
ah ok, it's when you are creating a new task, yes there is nothing for that at the moment.

There is a loading icon when you move a task on the board...

It's added on my todo list.

**Comment by orlov0562** [comment]
Thank you for your response!  Yep, it's when I create/modify project/task/description etc. 

Should I close this ticket for now or just leave it?

**Comment by fguillot** [comment]
Leave it open because it's synced to my own Kanboard instance :)

**Comment by fguillot** [comment]
It's now available in the development branch for submit buttons.
